### DISCUSSION TITLE
pdf.image module 'PIL.Image' has no attribute 'Resampling'

### DISCUSSION THREAD

**Discussion by sufyanAshraf** [py-pdf/fpdf2#454] [General]
pdf = FPDF()
pdf.add_page()
pdf.set_font("Times", 'B', size=20) 
pdf.cell(0, 10,"Report", ln=1, align='C')
pdf.set_font("Times", size=10)
pdf.image(image_url, 10, 8, 33) # error here
![Screenshot 2022-06-09 193533](https://user-images.githubusercontent.com/44521060/172873287-54b10eda-7895-4d26-a084-d96c3197f0d7.png)

**Comment by Lucas-C** [comment]
Hi @sufyanAshraf 

It's very likely that you need to upgrade your version of Pillow: `pip install --upgrade Pillow`
